browser-interface/packages/shared/comms/sagas.ts

```diff
@@ -394,13 +393,12 @@ function* handleCommsReconnectionInterval() {
       USER_AUTHENTICATED: take(USER_AUTHENTICATED),
       timeout: delay(1000)
     })
+    // TODO: Why are we not doing `if (reason === undefined) continue`?
```

Comment:
It should make sense to add the 'if (reason === undefined) continue'. If you want to add it will be ok. Or just leave the comment to thing about it in the future.

Reply:
Another PR!